benchmarks/benchmark_rank.py

```diff
@@ -2,17 +2,24 @@
 from skimage import img_as_ubyte
 from skimage.filters import rank
 from skimage.filters.rank import __all__ as all_rank_filters
-from skimage.morphology import grey, disk
+from skimage.filters.rank import __3Dfilters__ as all_3d_rank_filters
```

Comment:
```suggestion
from skimage.filters.rank import __3Dfilters as all_3d_rank_filters
```

match the name in `skimage/filters/rank/__init__.py`